sound/soc/intel/boards/cnl_rt700.c

```diff
@@ -148,6 +149,11 @@ SND_SOC_DAILINK_DEF(sdw0_pin,
 SND_SOC_DAILINK_DEF(sdw0_codec,
 	DAILINK_COMP_ARRAY(COMP_CODEC("sdw:0:25d:700:0:0", "rt700-aif1")));
 
+SND_SOC_DAILINK_DEF(sdw2_pin,
+	DAILINK_COMP_ARRAY(COMP_CPU("SDW2 Pin0")));
+SND_SOC_DAILINK_DEF(sdw2_codec,
+	DAILINK_COMP_ARRAY(COMP_CODEC("sdw:2:25d:1308:0:0", "rt1308-aif")));
+
```

Comment:
lets add sdw3 as well?

Reply:
@plbossart We don't have rt715 codec driver for now. So, the sound card will not probed if we add sdw3.